src/database/queries.rs

```diff
@@ -45,6 +45,17 @@ impl SeaOrmStore<StatusListToken> {
             .map_err(|e| RepositoryError::FindError(e.to_string()))
     }
 
+    pub async fn find_by_issuer(
```

Comment:
Consider giving this a more descriptive name that better reflects its behavior.
```suggestion
    pub async fn find_all_by_issuer(
```

Reply:
Okay